src/app/GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -317,7 +320,8 @@ GitRevision GetActualRevisionForRevision(GitRevision revision)
                                firstRev: null,
                                secondRev: selectedRev,
                                summary: $"{_grepSummaryPrefix}{_fileStatusDiffCalculatorInfo.GrepArguments} {GetDescriptionForRevision(selectedRev.ObjectId)}",
-                               statuses);
+                               statuses,
+                               iconName: nameof(Images.DiffR));
```

Comment:
```suggestion
                               iconName: nameof(Images.ViewFile));
```

![image](https://github.com/user-attachments/assets/7dedd3c0-984e-46e2-b047-49893b330629)

Reply:
🤦 of course